help/tquery.md

```diff
@@ -0,0 +1,52 @@
+# Tquery
+
+### Typy danych i operatory filtrów
+
+| typ          | null(5) | = | in   | cmp | like | select | sort |
+|--------------|---------|---|------|-----|------|--------|------|
+| bool         | *       | * | .    | .   | .    | *      | +    |
+| date         | *       | * | *    | *   | .    | *      | +    |
+| datetime     | *       | . | .    | *   | .    | *      | +    |
+| int          | *       | * | *    | *   | *    | *      | +    |
+| string       | *       | * | *    | *   | *    | *      | +    |
+| uuid         | *       | * | *    | .   | .    | *      | .    |
+| text         | *       | . | .    | .   | *    | *(8)   | .    |
+| --------(9)  |         |   |      |     |      |        |      |
+| dict(1)      | *       | * | *    | .   | .    | *      | *    |
+| object(2)    | *       | . | .    | .   | .    | *      | .    |
+| dict_list(3) | *       | * | *(7) | .   | .    | *      | .    |
+| list(4)      | *       | . | .    | .   | .    | *      | .    |
+
+- `null` - operator poprawny tylko dla typów nullable
+- `=` - porównanie z wartością tego samego typu, int-int, string-string, itp., brak możliwości porównania z null ani z
+  pustym stringiem
+- `==`(?) - porównanie typu binary, bardziej kosztowne dla sql'a, ale z zachowaniem wielkości liter, polskich liter,
+  białych znaków
```

Comment:
A zwykłe = ignoruje jakieś białe znaki? Mam nadzieję że nie chodzi o trimmowanie, bo ma go tu nie być, przynajmniej docelowo.

Reply:
dla = może powinienem dodać, że należy szukać po strimmowanej wartości, to wtedy ten case gdzie domyślne szukanie jest _pad_ nie będzie robiło różnicy